code/modules/hydroponics/seeds.dm

```diff
@@ -168,7 +168,8 @@
 			data = list("blood_type" = "O-")
 		if(rid == "nutriment" || rid == "vitamin")
 			// apple tastes of apple.
-			data = T.tastes

```

Comment:
#9150 Handles this one in a different way, im not 100% sure what data = T.tastes does and theirs a few things

Reply:
it's likely the same runtime as per your defib changes that had broken cyborg ones. better to just have a check anyway